RabbitMQ.Stream.Client/Consts.cs

```diff
@@ -8,11 +8,23 @@ namespace RabbitMQ.Stream.Client
 {
     internal static class Consts
     {
+        private static readonly Random s_random = new();
```

Comment:
I think you can drop this and just use `Random.Shared.Next()` on L22 and L27 :)

https://andrewlock.net/building-a-thread-safe-random-implementation-for-dotnet-framework/